agent/php_execute.c

```diff
@@ -331,15 +331,8 @@ typedef struct _nr_framework_table_t {
  */
 // clang-format: off
 static const nr_framework_table_t all_frameworks[] = {
-    /*
-     * Watch out:
-     *   cake1.2 and cake1.3 use a subdirectory named 'cake' (lower case)
-     *   cake2.0 and on use a subdirectory named 'Cake' (upper case file name)
-     */
-    {"CakePHP", "cakephp", NR_PSTR("cake/libs/object.php"), nr_cakephp_special_1,
-     nr_cakephp_enable_1, NR_FW_CAKEPHP},
-    {"CakePHP", "cakephp", NR_PSTR("cake/core/app.php"), nr_cakephp_special_2,
-     nr_cakephp_enable_2, NR_FW_CAKEPHP},
+    {"CakePHP", "cakephp", NR_PSTR("cakephp/src/core/functions.php"), 0,
+     nr_cakephp_enable, NR_FW_CAKEPHP},
```

Comment:
Does this file only exist on cakePHP 4+? There are many names/comments referencing CakePHP 4+, but as far as I can tell, this is the only place en/disabling our instrumentation. Since we are presumably not testing on version 3 and below, we probably want to ensure that our instrumentation is disabled for that.

Reply:
Tests for CakePHP 3.x have been added in our testing repo.

Reply:
What about 2.x and 1.x? Does this enable on those too? Does our instrumentation work/break for those? Or are they old enough (or PHP version incompatible) that we don't need to consider it?

Reply:
It does not enable for CakePHP 1.x/2.x because the detection file that we are using does not exist in CakePHP 1.x/2.x